index.html

```diff
@@ -1,70 +1,443 @@
+            <a href="#"
+              ><i
+                class="fa-brands fa-linkedin-in"
+                target="_blank"
+                aria-label="Linkedin"
+              ></i
+            ></a>
+          </li>
+          <li class="social-icon">
+            <a href="#"
+              ><i
+                class="fa-brands fa-angellist"
+                target="_blank"
+                aria-label="Angellist"
+              ></i
+            ></a>
+          </li>
+          <li class="social-icon">
+            <a href="#"
+              ><i
+                class="fa-brands fa-twitter"
+                target="_blank"
+                aria-label="Twitter"
+              ></i
+            ></a>
+          </li>
+          <li class="social-icon">
+            <a href="#"
+              ><i class="fa-solid fa-m" target="_blank" aria-label="Medium"></i
+            ></a>
+          </li>
+        </ul>
+      </section>
+      <div class="box3">
+        <p class="box3c">Get my resume</p>
+      </div>
+      <div class="skills">
+        <div class="frame1"></div>
+        <div class="frame2">
+          <div class="framel">
+            <h2 class="lan">Languages</h2>
+          </div>
+          <div class="framer">
+            <li class="soc">
+              <a class="soc1" href="#"
+                ><i class="fa-solid fa-chevron-up fa-rotate-180"></i
+              ></a>
+            </li>
+          </div>
+        </div>
+      </div>
+      <ul class="list">
+        <li class="jsfr">
+          <div class="jst">
+            <p class="jste">JavaScript</p>
+            <img class="im" src="./images/Ellipse 1.png" alt="Ellipse 1" />
+          </div>
+        </li>
+        <li class="jsfr">
+          <div class="jst">
+            <p class="jste">HTML</p>
+            <img
+              class="im"
+              src="./images/Ellipse 1 (1).png"
+              alt="Ellipse 1 (1)"
+            />
+          </div>
+        </li>
+        <li class="jsfr">
+          <div class="jst">
+            <p class="jste">CSS</p>
+            <img
+              class="im"
+              src="./images/Ellipse 1 (2).png"
+              alt="Ellipse 1 (2)"
+            />
+          </div>
+        </li>
+      </ul>
+
+      <div class="endtop">
+        <div class="endl">
+          <p class="framworks">Framworks</p>
+        </div>
+        <div class="endr">
+          <li class="soc">
+            <a class="soc1" href="#"
+              ><i class="fa-solid fa-chevron-up fa-rotate-90"></i
+            ></a>
+          </li>
+        </div>
+      </div>
+      <hr class="w" />
+      <div class="endtop">
+        <div class="endl">
+          <p class="framworks">Skills</p>
+        </div>
+        <div class="endr">
+          <li class="soc">
+            <a class="soc1" href="#"
+              ><i class="fa-solid fa-chevron-up fa-rotate-90"></i
+            ></a>
+          </li>
+        </div>
+      </div>
+      <hr class="w" />
```

Comment:
- [ ] The style of the Skills/Languages/Frameworks list is very good and similar to the template, but the HTML structure is a little bit off. Here we can see 2 lists, one major list which contains the Skills/Languages/Frameworks as list items, and inside the Languages item, we can see another list of the languages. Kindly wrap the major list with the `ul` tag and each item inside it with the `li` , and for the Languages list, wrap the languages again with the `ul` and each language wrap with the `li` tag.
 - 
<img width="1006" alt="image" src="https://user-images.githubusercontent.com/61919617/228623537-eb1b4229-5d75-4d3d-b99c-896a6a5ffbb2.png">